samples/nfc/writable_ndef_msg/boards/nrf54l15dk_nrf54l10_cpuapp.conf

```diff
@@ -0,0 +1,8 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+CONFIG_NVS=n
+CONFIG_ZMS=y
```

Comment:
What do you think to enable backend in Kconfig file?
```
config ZMS
	default y if (SOC_FLASH_NRF_RRAM || SOC_FLASH_NRF_MRAM)

config NVS
	default y if !(SOC_FLASH_NRF_RRAM || SOC_FLASH_NRF_MRAM)
```
This will eliminate platform dependencies and additional files.

Reply:
Good point!

Reply:
Updated as suggested